x-pack/functionbeat/function/telemetry/telemetry.go

```diff
@@ -0,0 +1,51 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package telemetry
+
+import (
+	"strconv"
+	"sync"
+
+	"github.com/elastic/beats/libbeat/monitoring"
+)
+
+// T is a telemetry instance
+type T interface {
+	AddTriggeredFunction(string, int64, int64)
+}
```

Comment:
just from the interface definition it is a little difficult to tell what the function arguments are supposed to mean.